test/multiverse/suites/unicorn/Envfile

```diff
@@ -0,0 +1,16 @@
+SUPPORTED_UNICORN_VERSIONS = [
```

Comment:
Are these the only supported versions, or is it just what we're going to be testing? Just wanted to clarify because of the name here confused me a little.

Reply:
Great question! These are the supported versions once we get to major release 9.0. I made the mistake of consulting the EOL Gem Spreadsheet instead of our public documentation for the versions here. We should really have something from the 4.x series right now to make sure we're reflecting the public docs!

Reply:
@tannalynn  - I've added 4.9.0 to the supported versions list. Strangely, it doesn't have a Ruby 3.0 restriction like Unicorn 5.3.0! 😅 

Please re-review when you can!